modules/ROOT/pages/export-to-exchange-task.adoc

```diff
@@ -5,30 +5,23 @@ endif::[]
 
 Anypoint Studio allows you to connect to Exchange to publish your project as a new asset.
 
-. Right click the project folder on your Package Explorer View.
-. Select Anypoint Platform and then Publish to Exchange.
-+
-If you don't have your Anypoint Platform credentials configured in Anypoint Studio, you'll need to provide your log in information before continuing.
-. Select your Anypoint Platform user and business group.
-+
+. Right-click your project's folder on your Package Explorer view.
+. Select *Anypoint Platform* > *Publish to Exchange*.
+. Select your Anypoint Platform user and business group. +
 Additionally you can click the "Add Account" button to use a new Anypoint Platform account.
 . Review the information about your project:
-* Group Id: This field is required. The group Id for your project is the Organization Id of your Anypoint Platform Organization.
-* Artifact Id: This field is required. The artifact Id is the name of your project without a version. By default, Studio uses the artifact Id configured in your pom.xml file.
-* Version: This field is required. The version of the asset you are publishing.
-. Select a project type.
-. Select the "Save updated project information to the pom.xml"  if you want to update your pom.xml file with the data you just provided.
-+
-[NOTE]
---
-If selected, the "Save updated project information to the pom.xml" option updates your local project's information with the data configured in the Publish to Exchange window. +
-If not, the project is deployed with the configured data, but your local project information is not updated.
---
-+
-. Click Finish.
+* *Group ID*: The group ID for your project is the Organization ID of your Anypoint Platform Organization. This field is required.
+* *Artifact ID*: The artifact ID is the name of your project without a version. By default, Studio uses the artifact ID configured in your pom.xml file. This field is required.
+* *Version*: The version of the asset you are publishing. This field is required.
+. Select a project type. +
+If you are publishing a new version of an already published asset, you cannot change the project type.
+. Select the *Save updated project information to the pom.xml*  if you want to update your pom.xml file with the data you just provided. +
+This option option updates your local project's information with the data configured in the `Publish to Exchange` window. If you do not select this option, the project is deployed with the configured data, but your local project information is not updated.
+. Select *Finish*. +
+
+You can follow the deployment progress in the Console View.
```

Comment:
```suggestion
You can follow deployment progress in the *Console* view.
```

Reply:
However, I think it makes more sense to follow the final step with something like:

After you select *Finish*, the publication of your project as a new asset in Exchange begins and cannot be cancelled. You can follow the progress of this publication process in the *Console* view.